README.rst

```diff
@@ -296,3 +296,39 @@ This project is maintained by **FOSSASIA**. See the AUTHORS file for a list of a
 .. _eventyay.com: https://eventyay.com
 .. _blog: https://blog.eventyay.com
 .. _Docker secrets: https://docs.docker.com/engine/swarm/secrets/
+
+
+
+
+
+
+
+Local Development Notes
+-----------------------
+
+Eventyay is a monorepo consisting of multiple services. New contributors should note:
+
+* There are multiple ``manage.py`` files. For the main backend, use:
+  
+  ``app/manage.py``
+
+* Eventyay currently requires **Django 5.2.x**. Newer Django versions may break local setup.
+
+* The default development configuration uses test domains (e.g. ``test.eventyay.com``).
+  For local development, create a local override file:
+
+  ``app/eventyay/config/eventyay.local.toml``
+
+  Example:
+
+  ::
+
+    site_url = 'http://localhost:8000'
```

Comment:
These values are default, no need to add again to the _eventyay.local.toml_ file. 
Only add the values you want to override. Check again the "How to override the configuration values" section.